connection.go

```diff
@@ -335,7 +335,11 @@ func (sc *snowflakeConn) ExecContext(
 		return sc.handleMultiExec(ctx, data.Data)
 	}
 	logger.Debug("DDL")
-	return driver.ResultNoRows, nil
+	return &snowflakeResult{
```

Comment:
I think we shouldn't change this, as this is a BCR. Client code may depend on returning `ResultNoRows`.

Reply:
`ResultNoRows` is a struct implementing `driver.Result` methods (`LastInsertId()`, `RowsAffected()`). These methods throw an error when used. `SnowflakeResult` is also an implementation of `driver.Result` with additional features thus I do not believe that it is a BCR.